src/containers/our-ecosystem/index.jsx

```diff
@@ -3,7 +3,7 @@ import clsx from "clsx";
 import PropTypes from "prop-types";
 // import Button from "@ui/button";
 import ReadMoreBtn from "@components/read-more-button";
-import Image from "next/image";
+import { Image } from "react-bootstrap";
```

Comment:
why importing 'Image' from react-bootstrap? it should be from "next/image"

Reply:
I was not guided to use manager.rumsan.com so i kept the image in public and need to use the react/bootstrap instead of next/image as the image were not rendering the as the width and height mentioned